assets/charts.html

```diff
@@ -0,0 +1,187 @@
+                    <img alt="GitHub closed issues" src="https://img.shields.io/github/issues-closed/Tkd-Alex/Twitch-Channel-Points-Miner-v2">
+                </a>
+                <a style="text-decoration: none;" href="https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2">
+                    <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/Tkd-Alex/Twitch-Channel-Points-Miner-v2" />
+                </a>
+            </div>
+        </div>
+        <br>
+
+        <div class="columns">
+            <div class="column is-10">
+                <div class="tabs">
+                    <ul>
+                        {% for streamer in streamers.split(',')|sort %}
+                            <li onClick="changeStreamer('{{ streamer }}', {{ loop.index }})"><a>{{ streamer.replace(".json", "")  }}</a></li>
+                        {% endfor %}
+                    </ul>
+                </div>
+            </div>
+            <div class="column" style="text-align: center">
+                <label class="checkbox">
+                    <input type="checkbox" checked="true" id="annotations">
+                    Annotations
+                </label>
+            </div>
+        </div>
+
+        <div id="chart"></div>
+
+        <hr>
+        <div style="text-align: center">If you want to help on this project, please leave a star 🌟 and share it with your friends! 😎 - A coffee is always a gesture of LOVE ❤️
+            <a href="https://www.buymeacoffee.com/tkdalex" target="_blank">
+                <img style="margin-bottom: -8px; margin-left: 10px;"  src="https://cdn.buymeacoffee.com/buttons/lato-yellow.png" alt="Buy Me A Coffee" height="5" width="138">
+            </a>
+        </div>
+    </div>
+</body>
+<script type="text/javascript">
+    // https://apexcharts.com/javascript-chart-demos/line-charts/zoomable-timeseries/
+    var options = {
+        series: [],
+        chart: {
+            type: 'area',
+            stacked: false,
+            height: 400,
+            zoom: {
+                type: 'x',
+                enabled: true,
+                autoScaleYaxis: true
+            }
+        },
+        dataLabels: {
+            enabled: false
+        },
+        stroke: {
+            curve: 'smooth',
+        },
+        markers: {
+            size: 0,
+        },
+        title: {
+            text: 'Channel points',
+            align: 'left'
+        },
+        fill: {
+            type: 'gradient',
+            gradient: {
+                shadeIntensity: 1,
+                inverseColors: false,
+                opacityFrom: 0.5,
+                opacityTo: 0,
+                stops: [0, 90, 100]
+            },
+        },
+        yaxis: {
+            title: {
+                text: 'Channel points'
+            },
+        },
+        xaxis: {
+            type: 'datetime'
+        },
```

Comment:
The times that are saved into the json file are the local datetime. Though by default the axis will interpret it as being UTC datetimes.

As my server has a timezone of CET (UTC+1), all the dates in the graph are shifted by an hour (13h, becomes 12h).

Either we need to save dates as UTC inside the json (which is in my opinion the best/safest). Or we can tell the axis to load them as local time (by assuming the tz of the browser is the same as the tz of the server):
```suggestion
        xaxis: {
            type: 'datetime',
            labels: {
                datetimeUTC: false
            }
        },
```